shepherd/templates/submit.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Code Submission</title>
+	    <link rel="stylesheet" type="text/css" href={{url_for( 'static', filename="PerkSelection/css/submit.css")}} media="screen" />
+  		<script type="text/javascript" src={{url_for( 'static', filename='socket.io.js' )}}></script>
+	    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
```

Comment:
Given how unreliable our field internet is, I would serve jQuery locally instead of relying on this file being cached.